src/api.raml

```diff
@@ -832,6 +832,44 @@ traits:
                                 description: |
                                     Demonstration of the list being empty (the first page will always return a valid result).
                                 value: !include samples/digest-list/v1/empty.json
+    post:
+        description: |
+            Add a new digest.
```

Comment:
could add request headers for authorization, but can either be `Authorization` (for the `api-gateway`) or `X-Consumer-Group` (probably wrong, for the backend service)